library-app/src/helpers/manageLocalStorage.tsx

```diff
@@ -0,0 +1,25 @@
+export interface LocalStorage {
+  AccessToken: string
+  RefreshToken: string
+  Expiration: string
+}
+
+const accessTokenKey = 'accessToken'
+const refreshTokenKey = 'refreshToken'
+const expirationKey = 'expiration'
+
+export const setLocalStorage = ({ AccessToken, RefreshToken, Expiration }: LocalStorage) => {
+  localStorage.setItem(accessTokenKey, AccessToken)
+  localStorage.setItem(refreshTokenKey, RefreshToken)
+  localStorage.setItem(expirationKey, Expiration)
```

Comment:
Alternatively, you can hold whole object in localstorage.setItem(JSON.stringify(object))

Reply:
getting would go like this: JSON.parse(localStorage.getItem('objectNameInStorage')).